src/containers/Review/ReviewWrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { Route, Switch } from 'react-router'
+import { Header } from 'semantic-ui-react'
+import { NoMatch } from '../../components'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { FullStructure } from '../../utils/types'
+
+const ReviewWrapper: React.FC = () => {
+  const {
+    match: { path },
+    query: { reviewId },
+  } = useRouter()
+
+  return (
+    <Switch>
+      <Route exact path={path}>
+        <ReviewHomeNEW />
+      </Route>
+      <Route exact path={`${path}/:reviewId`}>
+        <ReviewPageNEW reviewId={Number(reviewId)} />
```

Comment:
reviewId would be extracted further on, we would just need to pass though structure and reviewAssignments

Reply:
should be done as part of issue #378, but I can remove this reviewId from the props now.

Reply:
it's ok, we can do it in another issue